valhalla/thor/unidirectional_astar.h

```diff
@@ -21,24 +21,22 @@
 namespace valhalla {
 namespace thor {
 
+enum class AStarExpansionType { forward, reverse };
```

Comment:
can we move this to path algorithm and share it with bidirectional a*

Reply:
Done